.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -1,35 +1,13 @@
-## Status
+Fixes #
 
-Ready for review / Work in progress
-
-## Description of Changes
-
-Fixes #.
-
-Changes proposed in this pull request:
-
-## Testing
-
-If you made non-trivial code changes, include a test plan and validated it for this PR.
-
-## Deployment
-
-Any special considerations for deployment? Consider both:
-
-1. Upgrading existing pilot instances
-2. New installs
+## Test plan
+<!-- Delete this section if not applicable (e.g., some docs-only changes) -->
 
 ## Checklist
 
-### If you have made changes to the provisioning logic
-
-- [ ] All tests (`make test`) pass in `dom0`
```

Comment:
By removing this checkbox for tests, I assume the implication is that if all tests aren't passing then the PR wouldn't be ready for review in the first place - is that correct?

I think removing this is the right move, I just want to make sure we're not overlooking it.

Reply:
Yes; one related principle that Kunal suggested was that checklist items shouldn't duplicate stuff that CI takes care of for us. IIRC this checklist item pre-dates us having automated tests for `dom0`; now that we do, it seems reasonable to rely on them.

Reply:
(Of course, `dom0` tests are still a bit flaky in CI as the test run on this PR demonstrates, so this does assume that we'll need to diligent when we ignore them vs. re-run them.)

Reply:
Excellent, that principle makes my minimalist heart happy :)